Info.plist

```diff
@@ -5,17 +5,17 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
 	<key>CFBundleDisplayName</key>
-	<string>OpenRA</string>
+	<string>{MOD_NAME}</string>
 	<key>CFBundleExecutable</key>
 	<string>OpenRA</string>
 	<key>CFBundleIconFile</key>
 	<string>OpenRA.icns</string>
 	<key>CFBundleIdentifier</key>
-	<string>net.openra.launcher</string>
```

Comment:
People will probably want to change this entire value but for the sake of moving this work forward we can do that later.